CHANGES

```diff
@@ -81,6 +97,9 @@ Standard Library
 
 Plugins
 
+- The Ssreflect plugin is now distributed with Coq. Its documentation has
+  been integrated as a chapter of the reference manual. This chapter is
+  still imperfect so feedback is welcome.
```

Comment:
How about "preliminary" instead of "imperfect" (which makes it sound like it's badly-written)